packages/notifi-frontend-client/lib/client/NotifiFrontendClient.ts

```diff
@@ -73,48 +81,56 @@ export type WalletWithSignParams = Readonly<{
 
 export type WalletWithSignMessage =
   | Readonly<{
-      walletBlockchain: 'SOLANA';
-      walletPublicKey: string;
-      signMessage: Uint8SignMessageFunction;
-    }>
+    walletBlockchain: 'SOLANA';
+    walletPublicKey: string;
+    signMessage: Uint8SignMessageFunction;
+  }>
   | Readonly<{
-      walletBlockchain:
-        | 'ETHEREUM'
-        | 'POLYGON'
-        | 'ARBITRUM'
-        | 'AVALANCHE'
-        | 'BINANCE'
-        | 'OPTIMISM'
-        | 'ZKSYNC'
-        | 'BASE';
-
-      walletPublicKey: string;
-      signMessage: Uint8SignMessageFunction;
-    }>
+    walletBlockchain:
+    | 'ETHEREUM'
+    | 'POLYGON'
+    | 'ARBITRUM'
+    | 'AVALANCHE'
+    | 'BINANCE'
+    | 'OPTIMISM'
+    | 'ZKSYNC'
+    | 'BASE'
+    | 'INJECTIVE'
```

Comment:
Double confirm wether it is good to put injective here?
Seems that we also need to have `accountAddress` for INJECTIVE. don't we?

Reply:
Shall we also revert the Injective here??